### ISSUE TITLE
Exact coverage calculation fails in the self.coverage_and_or_pdf_scan method

### ISSUE THREAD

**Issue by jvaracarbonell** [thoglu/jammy_flows#6]
When [ `exact_coverage_calculation`](https://github.com/thoglu/jammy_flows/blob/4ca7aeccba04bc3c73d33f158f3f1dbea366381b/jammy_flows/main/default.py#L1972) is set to true, an error occurs because the number of pdf_evals obtained from [ `get_multiresolution_evals`](https://github.com/thoglu/jammy_flows/blob/4ca7aeccba04bc3c73d33f158f3f1dbea366381b/jammy_flows/main/default.py#L2145) does not correspond to a fixed number of pixels associated with a given nside in HEALPix. This results in an error,  [as seen.](https://github.com/thoglu/jammy_flows/blob/4ca7aeccba04bc3c73d33f158f3f1dbea366381b/jammy_flows/helper_fns/contours.py#L134)

**Comment by thoglu** [comment]
can you use a workaround for now? I have this function fixed in a version on my local machine, but it is not pushed yet

**Comment by jvaracarbonell** [comment]
Yes, I think I fixed it by parsing `sample_points=eval_positions` [here](https://github.com/thoglu/jammy_flows/blob/4ca7aeccba04bc3c73d33f158f3f1dbea366381b/jammy_flows/main/default.py#L2168). However, I now get a different error:  `AttributeError: module 'jammy_flows.helper_fns.contours' has no attribute 'find_closest_contour'`. Due to [this](https://github.com/thoglu/jammy_flows/blob/4ca7aeccba04bc3c73d33f158f3f1dbea366381b/jammy_flows/main/default.py#L2176). Do you know if that function might have a different name in the GitHub version?

**Comment by thoglu** [comment]
should be fixed in new release v1.1.0
